app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java

```diff
@@ -1,90 +1,62 @@
 package com.globant.scriptsapadea.ui.fragments;
 
-import android.content.Intent;
+import android.graphics.Bitmap;
 import android.os.Bundle;
-import android.provider.MediaStore;
 import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
 import android.widget.TextView;
 
 import com.globant.scriptsapadea.R;
-import com.globant.scriptsapadea.manager.ActivityResultEvent;
-import com.squareup.otto.Subscribe;
+import com.globant.scriptsapadea.utils.ImageRealPath;
+import com.squareup.picasso.Picasso;
+
+import java.io.File;
 
 /**
- * Created by leonel.mendez on 5/19/2015.
+ * Created by leonel.mendez on 6/11/2015.
  */
-public class PictureFragment extends BaseFragment implements View.OnClickListener {
-
-    private static final String PATIENT_NAME = "patientname";
+public class PictureFragment extends BaseFragment {
 
-    private static int GALLERY = 0x001;
-    private static int CAMERA = 0x010;
+    public static final String SCREENPLAY_IMAGE = "picture_image";
+    public static final String PICTURE_FROM_CAMERA = "picture_from_camera";
+    public static final String PATIENT_NAME = "patient_name";
 
-    public static PictureFragment newInstance(String patientName) {
-        PictureFragment fragment = new PictureFragment();
-        Bundle args = new Bundle();
-        args.putString(PATIENT_NAME, patientName);
-        fragment.setArguments(args);
-        return fragment;
+    public static PictureFragment newInstance(Bundle imageBundle){
+        PictureFragment pictureFragment = new PictureFragment();
+        pictureFragment.setArguments(imageBundle);
+        return pictureFragment;
     }
 
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_picture, container, false);
-
-        TextView txtPatientName = (TextView) view.findViewById(R.id.txt_patient_name);
-
-        if (getArguments() != null) {
-            String name = getArguments().getString(PATIENT_NAME);
```

Comment:
@leomendezg arguments should be taken here is case you need some data for a miss calculation before the view get ready.